docs/reference/stm/tarray.md

```diff
@@ -130,14 +130,15 @@ val foldSTMTArray: UIO[Int] = (for {
 
 ## Perform side-effect for TArray elements
 
-`foreach` is used for performing side-effect for each element in the array:
+`foreach` is used for performing an STM effect for each element in the array:
```

Comment:
Do the headings above these sections also need to be updated?

Reply:
ah... 😞

Reply:
Fixed. I kept the word effect but removed `side-`